### PR TITLE
[FAB-17303] Support new organization configuration field in channel config

### PR DESCRIPTION
**Pull request by caod123**
* PrivateDataImplicitCollectionConfig stores the configuration for
org-specific implicit collections

### CODE REVIEW

File: peer/configuration.proto [hunk 1]
```diff
@@ -32,3 +32,9 @@ message APIResource {
 message ACLs {
     map<string, APIResource> acls = 1;
 }
+
+// PrivateDataImplicitCollectionConfig stores the configuration for org-specific implicit collections
+message PrivateDataImplicitCollectionConfig {
```
Comment (jyellick) [inline peer/configuration.proto]: Not to bikeshed, but this stutters a bit.  We don't include `Config` in any of the other names.

It's `MSP`, not `MSPConfig`, it's `AnchorPeers` not `AnchorPeersConfig`, `BatchSize`, not `BatchSizeConfig` etc.

Maybe just `PrivateDataImplicitCollection` ?

Comment (caod123) [reply peer/configuration.proto]: I have no preference, @denyeart suggested the name originally. Thoughts?

Comment (caod123) [reply peer/configuration.proto]: Dave agreed, pushed

**Pull request merged**